frontend/pages/admin_v2.py

```diff
@@ -2,52 +2,50 @@
 import extra_streamlit_components as stx
 
 from datastore.db import Database
+
 db = Database()
 db.create_table()
 server, user = st.tabs(["Server Managment", "User Managment"])
 
+def create_dynamic_buttons(idx,status="start"):
+     for item in range(len(servers)):
+         count = idx*len(servers) + item
+         action_button = st.button(status, key=count)
+
+         if action_button:
+             st.toast(f'service has been {status}ed')
```

Comment:
Looks good!

I suggest to add `servers` as an argument of the function.

Now it implicitly reads it from the global namespace which may result in bugs.